Karl/karl.h

```diff
@@ -39,13 +39,18 @@ SOFTWARE.
 // The conventional wisdom is that Karl can start with both 1) and 2) missing. After one keepalive cycle,
 // which is under half a minute, it would regain the state of the fleet, as long as all keepalives go to it.
 
-// NOTE: Local `current_build.h` must be included before Karl/Claire headers.
+// NOTE(mzhurovich): Local `current_build.h` must be included before Karl/Claire headers.
+// NOTE(dkorolev): Why the hell? :-)
```

Comment:
What exactly is wrong with `current_build.h` being `#include`-d directly and unconditionally from both `karl.h` and `claire.h`?